src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -26,8 +26,10 @@ public class FindCommand extends Command {
             + " and displays them as a list with index numbers.\n"
             + "Example: " + COMMAND_WORD + " n/ Names \n"
             + "Example: " + COMMAND_WORD + " i/ Industries \n"
+            + "Example: " + COMMAND_WORD + " f/ B C"
             + "Example: " + COMMAND_WORD + " f/ Funding Stages \n"
-            + "Example: " + COMMAND_WORD + " f/ B C";;
+            + "Note that funding stage SEED is only searchable by entering S \n"
```

Comment:
Good catch, should we also explicitly specify that "SERIES A/B/C" is searchable using "A/B/C" too?